server/app/views/admin/programs/ProgramIndexViewV2.java

```diff
@@ -230,108 +335,74 @@ private static Instant extractLastUpdated(
   }
 
-  private Tag maybeRenderManageTranslationsLink(Optional<ProgramDefinition> draftProgram) {
-    if (draftProgram.isPresent()) {
-      String linkText = "Manage Translations →";
-      String linkDestination =
-          routes.AdminProgramTranslationsController.edit(
-                  draftProgram.get().id(), LocalizedStrings.DEFAULT_LOCALE.toLanguageTag())
-              .url();
-      return new LinkElement()
-          .setId("program-translations-link-" + draftProgram.get().id())
-          .setHref(linkDestination)
-          .setText(linkText)
-          .setStyles(Styles.MR_2)
-          .asAnchorText();
-    } else {
-      return div();
-    }
+  private Tag renderManageTranslationsLink(ProgramDefinition program) {
+    String linkDestination =
+        routes.AdminProgramTranslationsController.edit(
+                program.id(), LocalizedStrings.DEFAULT_LOCALE.toLanguageTag())
+            .url();
+    ContainerTag button =
+        ButtonUtils.makeSvgTextButton("Manage translations", Icons.LANGUAGE_SVG_PATH)
+            .withId("program-translations-link-" + program.id())
+            .withClass(AdminStyles.TERTIARY_BUTTON_STYLES);
+    return ButtonUtils.asLinkButton(button, linkDestination);
   }
 
   private Tag maybeRenderViewApplicationsLink(
-      Optional<ProgramDefinition> activeProgram, Optional<CiviFormProfile> userProfile) {
-    if (activeProgram.isPresent() && userProfile.isPresent()) {
-      boolean userIsAuthorized = true;
-      try {
-        userProfile.get().checkProgramAuthorization(activeProgram.get().adminName()).join();
-      } catch (CompletionException e) {
-        userIsAuthorized = false;
-      }
-      if (userIsAuthorized) {
-        String editLink =
-            routes.AdminApplicationController.index(
-                    activeProgram.get().id(), Optional.empty(), Optional.empty())
-                .url();
-
-        return new LinkElement()
-            .setId("program-view-apps-link-" + activeProgram.get().id())
-            .setHref(editLink)
-            .setText("Applications →")
-            .setStyles(Styles.MR_2)
-            .asAnchorText();
-      }
+      ProgramDefinition activeProgram, CiviFormProfile userProfile) {
+    boolean userIsAuthorized = true;
+    try {
+      userProfile.checkProgramAuthorization(activeProgram.adminName()).join();
+    } catch (CompletionException e) {
+      userIsAuthorized = false;
     }
-    return div();
+    if (userIsAuthorized) {
+      String editLink =
+          routes.AdminApplicationController.index(
+                  activeProgram.id(), Optional.empty(), Optional.empty())
+              .url();
+
+      ContainerTag button =
+          ButtonUtils.makeSvgTextButton("Applications", Icons.TEXT_SNIPPET_SVG_PATH)
+              .withId("program-view-apps-link-" + activeProgram.id())
+              .withClass(AdminStyles.TERTIARY_BUTTON_STYLES);
+      return ButtonUtils.asLinkButton(button, editLink);
+    }
+    return null;
```

Comment:
suggestion (blocking): please avoid returning null from methods and return an Optional instead